scripts/run_test.sh

```diff
@@ -0,0 +1,212 @@
+    echo "kube-addon-manager successfully running."
+}
+function kube_dns(){
+    until kubectl -n kube-system get pods -lk8s-app=kube-dns -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 3; kubectl get pods -n default; echo "waiting for kube-dns to be available"; done
+    echo "kube-dns successfully running."
+    echo ""
+}
+function helm_init(){
+    helm init
+    until kubectl -n kube-system get pods -l app="helm",name="tiller" -o jsonpath="$JSONPATH" 2>&1 | grep -q "Ready=True"; do sleep 3;echo "waiting for tiller pod to be available"; done
+    echo ""
+}
+function create_kubeplus(){
+    echo "Creating Kubeplus pod ..."
+    kubectl apply -f $PROJECT_HOME/deploy
+    timeout=60
+    sleep_time=3
+    loops=$((timeout/sleep_time))
+    count=0
+    result=$(kubectl -n default get pods -lapiserver="true" -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+
+    while ! [[ "$result" ]]; do
+        if [[ "$count" -gt $loops ]]; then
+            exit 1
+        fi
+        result=$(kubectl -n default get pods -lapiserver="true" -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+        echo $result
+        sleep $sleep_time
+        echo "Waiting for KubePlus deployment..."
+        kubectl get pods -n default
+        count=$((count + 1))
+    done
+    echo "Kubeplus pod successfully running..."
+    echo ""
+}
+function create_operator(){
+    echo "Creating postgres-operator ..."
+    kubectl create -f $PROJECT_HOME/examples/postgres/postgres-operator.yaml
+    timeout=240
+    sleep_time=3
+    loops=$((timeout/sleep_time))
+    count=0
+    result=$(kubectl -n default get pods -lapp=postgres-operator  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+    pod=`kubectl get pods | grep kubeplus | awk '{print $1'}`
+    while ! [[ "$result" ]]; do
+        if [[ "$count" -gt $loops ]]; then
+            echo "Unable to create a postgres operator."
+            echo ""
+            echo "Operator deployer logs: "
+            kubectl logs $pod operator-deployer
+            echo ""
+            echo "Operator manager logs: "
+            kubectl logs $pod operator-manager
+            exit 1
+        fi
+        result=$(kubectl -n default get pods -lapp=postgres-operator  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+        echo $result
+        sleep $sleep_time
+        echo "Waiting for postgres-operator..."
+        kubectl get pods -n default
+        echo ""
+        count=$((count + 1))
+    done
+    echo "Successfully deployed postgres-operator!"
+    echo ""
+}
+function create_postgres(){
+    echo "Creating postgres instance ..."
+    kubectl create -f $PROJECT_HOME/examples/postgres/postgres1.yaml
+    timeout=600
+    sleep_time=3
+    loops=$((timeout/sleep_time))
+    count=0
+    result=$(kubectl -n default get pods -lapp=postgres1  -o jsonpath="$JSONPATH" 2>&1 | grep "Ready=True")
+    while ! [[ "$result" ]]; do
+        if [[ "$count" -gt $loops ]]; then
+            exit 1
```

Comment:
Can you print postgres pods here -- similar to how you are doing it create_operator (lines 80-87)?

Reply:
Sure

Reply:
This is not correct.

You should do following:

pod=`kubectl get pods | grep -v postgres-operator | grep postgres | awk '{print $1}'`

kubectl logs $pod